templates/base.html

```diff
@@ -18,23 +17,26 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-sm-2">
-
       {% block sidebar %}
         {% include "includes/sidebar.html" %}
       {% endblock %}
-
+      <!-- Display username and login/logout links in sidebar -->
+      <div class="user-auth">
+        {% if user.is_authenticated %}
+          <p>Welcome, <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a>!</p>
```

Comment:
The URL tag `{% url 'taxi:driver-detail' user.id %}` assumes that 'driver-detail' is correctly defined in your `urls.py` with the namespace 'taxi'. Ensure that this URL name and namespace match your actual URL configuration.